libs/ledger-live-common/src/families/evm/config.ts

```diff
@@ -658,71 +658,80 @@ const evmConfig: CurrencyLiveConfigDefinition = {
     type: "object",
     default: {
       node: { type: "external", uri: "https://sepolia-rollup.arbitrum.io/rpc" },
-      explorer: { type: "etherscan", uri: "https://api-sepolia.arbiscan.io/api" },
+      explorer: {
+        type: "etherscan",
+        uri: "https://proxyetherscan.api.live.ledger.com/v2/api/421614",
+      },
     },
   },
   config_currency_polygon_zk_evm_testnet: {
     type: "object",
     default: {
       status: { type: "active" },
       node: { type: "external", uri: "https://rpc.public.zkevm-test.net" },
-      explorer: { type: "etherscan", uri: "https://api-testnet-zkevm.polygonscan.com/api" },
+      explorer: {
+        type: "etherscan",
+        uri: "https://proxyetherscan.api.live.ledger.com/v2/api/1442",
```

Comment:
Got a network error for this one 
![Screenshot 2025-01-20 at 16 18 00](https://github.com/user-attachments/assets/bda10052-13bc-461e-abe7-07046cd0323e)